about.html

```diff
@@ -0,0 +1,36 @@
+<head>
```

Comment:
there are various html errors. You need to upload each html page to an html validator to check for them - https://validator.w3.org/nu